ci/worker-files/worker-config

```diff
@@ -0,0 +1,2 @@
+MODES="0"
```

Comment:
Is there a reason this isn't `MODE`? ~Maybe even `EXECUTION_MODE` or `CI_EXECUTION_MODE` to be more descriptive?~ Or just make it RUN_MODE? Not sure if we have this or not, in the readme or somehwere where we talk about run_mode explain which level does what. 

Additionally, make run_mode 
0 - the easiest one (only linter), 
1 - (MODE 0) + speed_tester, 
2 - (MODE 1) + pktgen 
and so on. Currently I don't think they are in order?

Reply:
The reason it's modes is because worker.sh has the ability to parse and run multiple different modes. I figured it would be helpful for some nodes (like 33) to not have to run pktgen, and only run speed-tester. This could be a node-specific thing. Also, the reason linter is mode 1 right now is because it's not the default. The default is speed/pktgen unless it's an unauthenticated user (uncommon). Mode 0 right now runs speed tester and pktgen combined.

Reply:
Mode 1 and 0 both run linter, but mode 1, right now ignores pktgen/speed_tester